src/components/FormComponents/Sykmeldingsperiode/PeriodePicker.tsx

```diff
@@ -0,0 +1,139 @@
+import { useEffect, useState } from 'react';
+import { Controller, UseFormResetField, UseFormSetValue } from 'react-hook-form';
+import { Button } from '@navikt/ds-react';
+// TODO: remove beta version when DatePicker is published
+import { UNSAFE_DatePicker, UNSAFE_useRangeDatepicker } from 'ds-react-datepicker';
+
+import { SykmeldingFormValues } from '../../Sykmelding/SykmeldingForm';
+import FieldError from '../FieldError/FieldError';
+
+import styles from './PeriodePicker.module.css';
+
+type DateRange = {
+    from: Date | undefined;
+    to: Date | undefined;
+};
+
+type FomName = `periode.${number}.fom`;
+type TomName = `periode.${number}.tom`;
+
+export interface PeriodePickerProps {
+    index: number;
+    setValue: UseFormSetValue<SykmeldingFormValues>;
+    resetField: UseFormResetField<SykmeldingFormValues>;
+}
+
+function PeriodePicker({ index, setValue, resetField }: PeriodePickerProps): JSX.Element {
+    const [dates, setDates] = useState<DateRange>({ from: undefined, to: undefined });
```

Comment:
Blir det vanskelig å bruke `useController` her istedenfor? Da slipper vi å duplisere staten, også slipper vi å sende inn setValue og resetField tror jeg. :thinking: Også slipper man de to Controllerene under per DatePicker